build_projects/dotnet-host-build/build.ps1

```diff
@@ -8,7 +8,7 @@ param(
     [string]$Architecture="x64",
     [string]$TargetArch="",
     [string]$ToolsetDir="",
-    [string]$Framework="netcoreapp1.0",
+    [string]$Framework="netcoreapp1.1",
```

Comment:
`build.sh` doesn't have a parameter like this, actually. This is only used to set the "TARGETFRAMEWORK" environment variable (which is used in PrepareTargets.cs), not for anything else. On line 122, we always publish the build projects for `netcoreapp1.0` (this variable doesn't control that).